packages/web3/test/black_box/test/fixtures/system_tests_utils.ts

```diff
@@ -0,0 +1,473 @@
+/*
```

Comment:
Was receiving a modules error for this file in blackbox testing after the CJS/ESM PR so we include it in the blackbox tests now

Reply:
Is it possible to use link to the file here?

Reply:
I don't believe so, the blackbox tests cannot find the module when I symlink. Talked with junaid and he said its okay to duplicate. Maybe i should create an issue about this?
This is the [error](https://github.com/web3/web3.js/actions/runs/4625519650/jobs/8181477873)
```
Cannot find module 'web3/test/shared_fixtures/system_tests_utils' from 'test/web3-eth-accounts/hashMessage.test.ts'
web3:       18 | // the test takes a long time to run afterAll
web3:       19 | import Web3 from 'web3';
web3:     > 20 | import {
web3:          | ^
web3:       21 | 	closeOpenConnection,
web3:       22 | 	getSystemTestProvider,
web3:       23 | 	isWs,
web3:       at Resolver._throwModNotFoundError (node_modules/jest-resolve/build/resolver.js:491:11)
web3:       at Object.<anonymous> (test/web3-eth-accounts/hashMessage.test.ts:20:1)
```

Reply:
yes will fix